src/stats/show_stats.py

```diff
@@ -37,6 +42,16 @@ async def show_stats(ctx, stat_id):
 
     survey_question_id, stat_type, stat_title = stats_options[stat_id]
 
+    if cached_stat := db.get_show_stats_updated_at_with_url_or_none(
+        stat_id
+    ):
+        updated_at, url = cached_stat
+        if not is_graph_expired(updated_at):
+            await send_graph_to_channel(ctx, url, stat_title)
+            return
+        else:
```

Comment:
`else` is IMHO redundant.

Reply:
I will leave it there for better readibility